lms/djangoapps/grades/models.py

```diff
@@ -396,17 +398,21 @@ class Meta(object):
     percent_grade = models.FloatField(blank=False)
     letter_grade = models.CharField(u'Letter grade for course', blank=False, max_length=255)
 
+    # Information related to course completion
+    date_passed = models.DateTimeField(u'Date learner earned a passing grade', blank=True, null=True)
```

Comment:
Wording: Should this be `passed_timestamp` instead since it's not just the date?  Making it consistent with the wording for `course_edited_timestamp`.